docs/tutorial_langcheckchat.md

```diff
@@ -0,0 +1,125 @@
+# LangCheckChat: a Q&A system over LangCheck Docs that Auto-Evaluates itself
+
+## Introduction
+
+In this tutorial, we will explore how to build a very simple RAG system (which we call LangCheckChat) that allows you to ask questions about the [LangCheck documentation](https://langcheck.readthedocs.io/en/latest/). Then, we will explore how we can use LangCheck metrics to evaluate the system’s performance, and show those results to the user after each response.
+
+[LangCheckChat](https://github.com/citadel-ai/langcheckchat) is fully open source, so please follow along by trying it yourself if you can!
+
```

Comment:
I think it's good to have a `## Video Demo` section here with the demo video to capture people's attention early

Reply:
Pushed a commit that installs a sphinx extension so we can embed YouTube videos. Do you have an account for uploading the video?

We can also use https://github.com/sphinx-contrib/video but don't want to include the .mp4 inside the repo and I'm not sure how to store static files in GitHub

Reply:
hmm is what I did for the demo page in langcheckchat not good? https://github.com/citadel-ai/langcheckchat?tab=readme-ov-file#demo

I just uploaded as a github asset

Reply:
This markdown file is not rendered by GitHub, it's converted by Sphinx into HTML. So to embed a raw mp4 file, we need to use https://github.com/sphinx-contrib/video

But to do this we need to point to the mp4 file in this repo, or have a URL that ends in .mp4. 

The URL in https://github.com/citadel-ai/langcheckchat?tab=readme-ov-file#demo doesn't end in .mp4 so I couldn't use it directly (I'm also not sure how you uploaded it since it's not a file in the repo)

Reply:
Ahhh I see. In langcheckchat I just did a drag and drop, similar to how you can upload videos to PR descriptions.

Do we have a company YouTube account that we can use?

(Also, OOC why is uploading the mp4 file to the repo a problem (assuming we don't hit the large file limit)?)

Reply:
Ah just remembered we have a Citadel AI Vimeo account! I uploaded it there. The video player is a lot cleaner than YouTube too

(The video file is 50MB so would rather not keep it in the core repo unless there's no other way)

Reply:
oh cool, thanks!! so just to confirm, the video will show up in readthedocs but is not expected to render in GitHub? (This is what I'm seeing right now)
<img width="1087" alt="image" src="https://github.com/citadel-ai/langcheck/assets/107823399/04e6fc06-aca3-4b7c-814a-b0c9ba8b8b1e">

Reply:
Yep, it should show up on readthedocs but not in GitHub. You can generate the readthedocs UI locally by running step 2 and 3 here: https://langcheck.readthedocs.io/en/latest/contributing.html#documentation

Reply:
thanks! checked that it's showing up in readthedocs

Is it fine for it to not render properly in GitHub though? I guess we already have some sections that render better in readthedocs

Reply:
Yeah, I think the docs in GitHub are raw files and aren't meant to be read directly. (e.g. most of the docs/ folder are .rst files which don't contain any information before they're auto-populated when Sphinx converts them into HTML)